librz/core/cmd_descs/cmd_egg.yaml

```diff
@@ -0,0 +1,71 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_egg
+commands:
+  - name: g
+    cname: egg_compile
+    summary: Compile the shellcode
+    args:
+      - name: file
+        type: RZ_CMD_ARG_TYPE_FILE
+        optional: true
+  - name: gc
+    cname: egg_config
+    summary: Get/Set config option for shellcode / List all config options
+    args_str: " <key>[=<val|?>] [<key>[=<val|?>] ...]]"
+    args:
+      - name: key=value
+        type: RZ_CMD_ARG_TYPE_STRING
+        flags: RZ_CMD_ARG_FLAG_ARRAY
+        optional: true
+    details:
+      - name: Examples
+        entries:
+          - text: gc
+            arg_str: " egg.encoder"
+            comment: "Show current value of config variable `egg.encoder`"
+          - text: gc
+            arg_str: " egg.encoder=xor"
+            comment: "Set config variable `egg.encoder` to `xor`"
+  - name: gl
+    cname: egg_list_plugins
+    summary: List shellcode and encoder plugins
+    args: []
+  - name: gs
+    cname: egg_syscall
+    summary: Compile syscall "name(args)"
+    args:
+      - name: name
+        type: RZ_CMD_ARG_TYPE_STRING
+      - name: args
+        type: RZ_CMD_ARG_TYPE_STRING
+        optional: true
```

Comment:
shall this be an ARRAY?

Reply:
There is no `RZ_CMD_ARG_TYPE_ARRAY` exists.

Reply:
see `flags` in cmd_descs.yaml . RZ_CMD_ARG_FLAG_ARRAY

Reply:
Well, it's not flags, it's syscall arguments. Not sure if it's appropriate here.

Reply:
yes, but it's more than one. If you set `flags: RZ_CMD_ARG_FLAG_ARRAY` for the argument, RzCmd will know that it's an array of elements. Grep for it in the code, you'll see.

Reply:
Done.